gino/ext/tornado.py

```diff
@@ -272,45 +202,33 @@ class Model(__CRUDModel, TornadoModelMixin, metaclass=__ModelType):
         kwargs.setdefault('strategy', 'tornado')
         return await super().set_bind(bind, loop=loop, **kwargs)
 
-
-class Application(tornado.web.Application):
-    """
-    Base application that provides access to the database object and defines
-    a convenient method for initializing all the database-related stuff.
-
-    """
-
-    #: The database object associated with this application.
-    #: Use :py:meth:`~.late_init()` to init this or set it manually.
-    db = None
-
-    #: If ``True``, enables ``GinoRequestHandler`` to create lazy connections.
-    #:
-    #: See :py:attr:`~.GinoRequestHandler.use_connection_for_request`
-    #: for more info.
-    use_connection_for_request = True
-
-    async def late_init(self, db: Gino, *, loop=None, options=_options):
+    async def init_app(self, app, *, loop=None, dsn='', driver='asyncpg',
```

Comment:
I'd go further and removed `app` argument completely (and moved `db` assignment to the `init_db` function).

Reply:
I understand you'd like it to be less coupled, but I want those who don't want to subclass Application also have an option, otherwise there is no point to have `init_app` method at all.

Reply:
Yes, this makes sense.